apps/assistants/views.py

```diff
@@ -124,6 +126,13 @@ class EditOpenAiAssistant(BaseOpenAiAssistantView, UpdateView):
     button_text = "Update"
     permission_required = "assistants.change_openaiassistant"
 
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        if are_files_in_sync_with_openai(self.object):
```

Comment:
This should call `is_synced_with_openai` instead?

Reply:
Can this call not be bundled with the `check_sync_status` view?

Reply:
bundled here: https://github.com/dimagi/open-chat-studio/pull/774/commits/17d212d6230f6e0d322a7881ff4a740479e591aa